qemu/tests/cfg/slof_hugepage.cfg

```diff
@@ -0,0 +1,38 @@
+- slof_hugepage: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = slof_hugepage
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
```

Comment:
`login_timeout` defaults to 240, the re-assignment is redundant.

Reply:
thanks, have updated.